csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1779,95 +2023,104 @@ CSAF documents are based on JSON, thus the security considerations of [RFC8259]
 * To reduce the risk posed by possibly malicious CSAF files that do contain arbitrary HTML (including, for example, javascript: links), CSAF consumers SHALL either disable HTML processing (for example, by using an option such as the --safe option in the cmark Markdown processor) or run the resulting HTML through an HTML sanitizer.
 CSAF consumers that are not prepared to deal with the security implications of formatted messages SHALL NOT attempt to render them and SHALL instead fall back to the corresponding plain text messages.
 
+# 5 Conformance
 
-(Note: OASIS strongly recommends that Technical Committees consider issues that might affect safety, security, privacy, and/or data protection in implementations of their specification and document them for implementers and adopters. For some purposes, you may find it required, e.g. if you apply for IANA registration.
-
-While it may not be immediately obvious how your specification might make systems vulnerable to attack, most specifications, because they involve communications between systems, message formats, or system settings, open potential channels for exploit. For example, IETF [[RFC3552](#rfc3552)] lists “eavesdropping, replay, message insertion, deletion, modification, and man-in-the-middle” as well as potential denial of service attacks as threats that must be considered and, if appropriate, addressed in IETF RFCs.
-
-In addition to considering and describing foreseeable risks, this section should include guidance on how implementers and adopters can protect against these risks.
-
-We encourage editors and TC members concerned with this subject to read _Guidelines for Writing RFC Text on Security Considerations_, IETF [[RFC3552](#rfc3552)], for more information.
+In the only subsection of this section, the conformance targets and clauses are listed.
+The clauses matching the targets one to one, are listed in separate sub-subsections of the targets listing subsection.
 
-Remove this note before submitting for publication.)
+Informative Comments:
 
--------
+>The order in which targets, and their corresponding clauses appear is somewhat arbitrarily as there is no natural order on such diverse roles participating in the document exchanging ecosystem.
+>
+>Except for the target **CSAF document** all other 12 targets span a taxonomy of the complex CSAF ecosystems existing in and between diverse security advisory generating, sharing, and consuming communities.
+>
+>In any case, there are no capabilities organized in increasing quality levels for targets because the security advisory sharing communities follow the chain link model.
+>Instead, a single minimum capability level for every target is given to maintain important goals of providing a common framework for security advisories:
+>* Fast production, sharing, and actionable consummation of security advisories
+>* Consistent end to end automation through collaborating and not competing "actors"
```

Comment:
I'm not sure what "competing actors" refers to in this context?

Reply:
Maybe collaborating is good on its own.